fullstory.go

```diff
@@ -136,3 +136,16 @@ func (c *Client) ExportData(id int) (ExportData, error) {
 
 	return c.doReq(req)
 }
+
+func (c *Client) ExportDataWithCustomUa(id int, useragent string) (ExportData, error) {
```

Comment:
Do we envision wanting to set a custom user agent or other headers for this function and for the other client functions, too, in the future? Maybe the following would be a better design to update the existing function to take a broad modify request param:

```go
func ExportData(id int, ...modifyReq: func(r http.*Request)) (ExportData, error) { ... }
```

And callers can do:

```go
_, err := ExportData(id, WithUserAgent("foo"))

func WithUserAgent(ua string) func(r *http.Request) {
        return func(r *http.Request) { 
                r.Header.Set("User-Agent", ua)
        }
}

// Other functions such as WithCacheControl, etc.
```